### ISSUE TITLE
Discrepancy between ARIA Spec and Authoring for TABS

### ISSUE THREAD

**Issue by LaurenceRLewis** [w3c/aria#1646]
In the ARIA 1.2 Spec for [Tabs](https://www.w3.org/TR/wai-aria-practices-1.2/#wai-aria-roles-states-and-properties-21) it states:

"_The active tab element has the state aria-selected set to true and all other tab elements have it set to false._"

However in the ARIA Spec for [Tablist](https://www.w3.org/TR/wai-aria-1.2/#tablist) it only refers to aria-expanded. _"authors SHOULD ensure each visible tabpanel has its aria-expanded attribute set to true, and that the remaining hidden tabpanel elements have their aria-expanded attributes set to false._"

This seems like a bit of a discrepancy to me.

**Comment by scottaohara** [comment]
you're comparing the ARIA practices example of how to implement a single-select tab widget with the ARIA spec that outlines when to use `aria-expanded` in the situation where someone has created a tab widget where multiple tabpanels might be shown.

What I did notice here is the reference in the aria spec to use of `aria-expanded` on the `tabpanel`.  I thought this was resolved with https://github.com/w3c/aria/pull/1403.   Did we miss something or did this PR get overwritten?
